actions.go

```diff
@@ -48,7 +48,8 @@ type ActionSpec struct {
 // ValidateParams validates the passed params map against the given ActionSpec
 // and returns any error encountered.
 // Usage:
-//   err := ch.Actions().ActionSpecs["snapshot"].ValidateParams(someMap)
+//
+//	err := ch.Actions().ActionSpecs["snapshot"].ValidateParams(someMap)
```

Comment:
Why is this a change? Looks like spacing or something

Reply:
Removed, I added all the linters from the CI to my Goland and have them set to rewrite the files, I should probs change that.